doc/sphinx/changes.rst

```diff
@@ -8,6 +8,477 @@ Recent changes
+^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  The ``-compat`` :ref:`command line option <command-line-options>`
+  now raises a warning rather than an error when the compatibility file
+  doesn't exist. This enables easier use of the compat mechanism with
+  versions where the compatibility file doesn't exist yet
+  (`#19370 <https://github.com/coq/coq/pull/19370>`_,
+  by Pierre Roux).
+- **Changed:**
+  `coq_makefile` generated makefiles only install plugin `.cmxs` files in findlib locations
+  and stop putting a copy in `user-contrib` (the copy should be useless after the removal of plugin legacy loading)
+  (`#19841 <https://github.com/coq/coq/pull/19841>`_,
+  by Gaëtan Gilbert).
+- **Removed:**
+  `coqdep` flag `-m` (it was used through `coq_makefile`)
+  (`#19863 <https://github.com/coq/coq/pull/19863>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  :ref:`command line option <command-line-options>` ``-compat-from``
+  to enable writing compatibility files for libraries similarly to the
+  ``-compat`` option for Rocq
+  (`#19370 <https://github.com/coq/coq/pull/19370>`_,
+  by Pierre Roux).
+- **Added:**
+  The ``-compat`` :ref:`command line option <command-line-options>`
+  now silences deprecation warnings that were introduced since the
+  given version
+  (`#19370 <https://github.com/coq/coq/pull/19370>`_,
+  by Pierre Roux).
+
+RocqIDE
+^^^^^^^
+- **Changed:**
+  Improved Preferences dialog: larger margins,
+  tree of categories, sections in the categories,
+  spin buttons for numbers, preservation of the last
+  selected category, and more
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
+- **Fixed:**
+  All preferences are now applied after clicking Apply or OK
+  rather than immediately
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
+- **Fixed:**
+  Changing the allowed modifiers in the Shortcuts panel of
+  the Preferences dialog now immediately updates the available
+  modifiers for the listed items
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
+- **Added:**
+  Preference setting for unjustified conclusions background color
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
+- **Changed:**
+  CoqIDE is renamed to RocqIDE (the auxiliary binary `coqidetop` is not renamed)
+  (`#20036 <https://github.com/coq/coq/pull/20036>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  Warnings are now included in the Errors panel
+  (`#19188 <https://github.com/coq/coq/pull/19188>`_,
+  by Jim Fehrle).
+- **Fixed:**
+  Changing the position of buffer names (top, left,
+  bottom or right) no longer needs a restart
+  (`#19166 <https://github.com/coq/coq/pull/19166>`_,
+  by Sylvain Chiron).
+- **Added:**
+  Document tabs are now reorderable
+  (`#19166 <https://github.com/coq/coq/pull/19166>`_,
+  by Sylvain Chiron).
+
+Standard library
+^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Stdlib moved to its own repository, look for Stdlib own changelog
```

Comment:
```suggestion
  Stdlib moved to its own repository, look for Stdlib own changelog
```
I guess we should add a link to refman-stdlib here @Zimmi48 or @mattam82 would you know what the exact link would be with the new website?

Reply:
```suggestion
  Stdlib moved to its own repository, look for `Stdlib own changelog <https://rocq-prover.org/doc/v9.0/refman-stdlib/changes.html>`_
```